src/server/routes.ts

```diff
@@ -14,15 +14,27 @@ export interface Page {
   component: rt.ComponentType<PageProps>;
 }
 
-export interface ApiRouteModule {
-  default: router.MatchHandler;
-}
+export type ApiRouteModule = {
+  [
+    K in
+      | "default"
+      | "GET"
+      | "HEAD"
+      | "POST"
+      | "PUT"
+      | "DELETE"
+      | "CONNECT"
```

Comment:
Remove `CONNECT`. Deno native http / deploy will likeky block that method soon :-)

Reply:
Actually lets remove `TRACE` too. Not useful and is blocked by `fetch`.